test/jdk/java/awt/dnd/DnDRemoveFocusOwnerCrashTest.java

```diff
@@ -80,32 +83,20 @@ public static void main(String[] args) throws Exception {
                 frame.pack();
                 frame.setVisible(true);
 
-                try {
-                    robot.delay(FRAME_ACTIVATION_TIMEOUT);
-                } catch (Exception e) {
-                    e.printStackTrace();
-                    throw new RuntimeException("The test failed.");
-                }
-
-                Point p = dragSourceButton.getLocationOnScreen();
+                p = dragSourceButton.getLocationOnScreen();
```

Comment:
Getting the location just after `setVisible` on EDT may give an inaccurate result(especially if the tested system is slow, and decided to move the window somewhere else).

I suggest to wait as usual before getting the location
```
robot.waitForIdle();
robot.delay(500); // or FRAME_ACTIVATION_TIMEOUT
```

Reply:
Good idea. Added.